### ISSUE TITLE
Incorrect information for colIdeaStats_Raw collection ?

### ISSUE THREAD

**Issue by maheshkhiste** [MicrosoftDocs/powerapps-docs#2871]
I wanted to extend /customize the Employee ideas app so I started going through the documentation and stumble upon OnStart collections section where it mentions all of the collections which are initialized but when i checked the app it doesn't show up in the App start section. Following is the App start section from the Ideas App.

As App is not giving error ,it means these collections are defined somewhere but I don't know how I can trace back where its defined.

Following is the code from Ideas app App onstart :
/**READ ME START**
Welcome to the app!
Please note that modifying anything in this app may result in unexpected behavior. Please do so, at your own risk. As of this writing, the app has been documented in minimal fashion with annotations in functions, and the Documentation linked on the Help screen.
In subsequent releases, this app will be updated to include helpful annotations and documentation to support users in customizing the app, and understanding the patterns/functions included here within.
**READ ME END**/
// App Loading
Set(
    gblAppLoaded,
    false
);
// Localization
// User Language
Set(
    gblUserLanguage,
    Switch(
        Left(
            Language(),
            2
        ),
        "de",
        "de-DE",
        "en",
        "en-US",
        "es",
        "es-ES",
        "fr",
        "fr-FR",
        "it",
        "it-IT",
        "ja",
        "ja-JP",
        "nl",
        "nl-NL",
        "pt",
        "pt-BR",
        // Default
        "en-US"
    )
);
// Build Localization Collection, with User Language
ClearCollect(
    colLocalization,
    Filter(
        staticLocalizations,
        LanguageTag = gblUserLanguage
    )
);
Collect(
    colLocalization,
    Filter(
        staticLocalizations_Common,
        LanguageTag = gblUserLanguage
    )
);
// Params
Concurrent(
    Set(
        gblParamTeamId,
        Param("groupId")
    ),
    Set(
        gblParamChannelId,
        Param("channelId")
    ),
    Set(
        gblIsHostClientWeb,
        Param("hostClientType") = "web"
    )
);

// App Loaded (Correct Context)
Set(
    gblAppLoaded,
    true
);

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b8374c88-e3e4-2a10-f4d7-5202b6346ffe
* Version Independent ID: a8c65da7-1464-1e23-bbc8-4ab8a2bc6277
* Content: [Understand Employee ideas sample app architecture - Power Apps](https://docs.microsoft.com/en-us/powerapps/teams/employee-ideas-architecture)
* Content Source: [powerapps-docs/teams/employee-ideas-architecture.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/teams/employee-ideas-architecture.md)
* Service: **powerapps**
* Sub-service: **teams**
* GitHub Login: @josephshum
* Microsoft Alias: **jshum**

**Comment by maheshkhiste** [comment]
Hello @tapanm-MSFT  @bishalgoswami , is there any update on this issue ? 
I checked with couple of other community experts and they also see this an issue. 
Any update is appreciated

**Comment by tapanm-MSFT** [comment]
@maheshkhiste thank you for reaching out. This is in progress and documentation will be updated based on the internal discussions. In the meantime, if you have a business blocker due to being unable to customize the app, please reach out using a [support ticket](https://docs.microsoft.com/power-platform/admin/get-help-support) for help and investigation to get unblocked.

**Comment by tapanm-MSFT** [comment]
The sample app templates for Power Apps in Teams are now available in the [Power Apps Teams Sample App Templates repository](https://github.com/microsoft/teams-powerapps-app-templates). Please review the latest version of the [app templates and documentation](https://github.com/microsoft/teams-powerapps-app-templates). If the problem persists, [create a new issue](https://github.com/microsoft/teams-powerapps-app-templates/issues).
